scripts/spidhub-fargate/environments/prod/storage/config/spid-saml-check/v1/configvolume/spid_users.json

```diff
@@ -0,0 +1,562 @@
+		"mobilePhone": "3466433463",
+		"email": "laPulzelladOrléans@yahoo.fr",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "fieramosca",
+		"password": "password123",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-007",
+		"familyName": "Fieramosca",
+		"name": "Ettore",
+		"gender": "M",
+		"dateOfBirth": "1976-08-06",
+		"countyOfBirth": "CE",
+		"placeOfBirth": "B715",
+		"idCard": "patente Z1Z111111Z motorizzazioneBarletta 2017-03-09 2027-08-06",
+		"expirationDate": "2019-08-01",
+		"fiscalNumber": "TINIT-FRMTTR76M06B715E",
+		"mobilePhone": "3323323323",
+		"email": "trediciitalianicontrotredicifrancesi@hotmail.com",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "cleopatra",
+		"password": "password123",
+		"spidLevel": 2,
+
+		"spidCode": "SPID-008",
+		"familyName": "Filopatore",
+		"name": "Cleopatra Tea",
+		"gender": "F",
+		"dateOfBirth": "1969-01-25",
+		"countyOfBirth": "EE",
+		"placeOfBirth": "Z336",
+		"idCard": "passaporto SH555555 ambasciataEgitto 2015-05-25 2025-01-25",
+		"expirationDate": "2019-05-21",
+		"fiscalNumber": "TINIT-FLPCPT69A65Z336P",
+		"mobilePhone": "3805555555",
+		"email": "ilcobranonèunserpente@gmail.it",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "marcopolo",
+		"password": "password123",
+		"spidLevel": 3,
+
+		"spidCode": "SPID-009",
+		"familyName": "Polo",
+		"name": "Marco",
+		"gender": "M",
+		"dateOfBirth": "2001-09-30",
+		"countyOfBirth": "VE",
+		"placeOfBirth": "L736",
+		"idCard": "cartaIdentita AZ765432 comuneVenezia 2015-03-12 2025-09-30",
+		"expirationDate": "2019-10-01",
+		"fiscalNumber": "TINIT-PLOMRC01P30L736Y",
+		"mobilePhone": "3320606060",
+		"email": "ilmilione@virgilio.it",
+		"digitalAddress": "",
+		"ivaCode": "",
+		"companyName": "",
+		"registeredOffice": "",
+		"companyFiscalNumber": "",
+		"address": "Via Listz 21 00144 Roma",
+		"domicileStreetAddress": "Via Listz 21",
+		"domicilePostalCode": "00144",
+		"domicileMunicipality": "Roma",
+		"domicileProvince": "RM",
+		"domicileNation": "IT"
+	},
+	{
+		"username": "lucia.mondella",
+		"password": "password123",
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)